### PR TITLE
HDDS-7186. Add support for Rocksdb user info LOG configure.

### PR DESCRIPTION
**Pull request by ChenSammi**
https://issues.apache.org/jira/browse/HDDS-7186 

Tested in local one node cluster.

### CODE REVIEW

File: hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/DatanodeConfiguration.java [hunk 1]
```diff
@@ -72,6 +73,16 @@ public class DatanodeConfiguration {
       Duration.ofMinutes(10).toMillis();
 
   static final boolean CONTAINER_SCHEMA_V3_ENABLED_DEFAULT = false;
+  static final long ROCKSDB_LOG_MAX_FILE_SIZE_DEFAULT = 8 * 1024 * 1024;
```
Comment (kerneltime) [inline hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/DatanodeConfiguration.java:76]:
```suggestion
  static final long ROCKSDB_LOG_MAX_FILE_SIZE_BYTES_DEFAULT = 8 * 1024 * 1024;
```

File: hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/DatanodeConfiguration.java [hunk 2]
```diff
@@ -72,6 +73,16 @@ public class DatanodeConfiguration {
...
+  static final int ROCKSDB_LOG_MAX_FILE_NUM_DEFAULT = 100;
+  static final long ROCKSDB_DELETE_OBSOLETE_FILES_PERIOD_DEFAULT =
```
Comment (kerneltime) [inline hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/DatanodeConfiguration.java:78]: Guessing this is micro seconds
```suggestion
  static final long ROCKSDB_DELETE_OBSOLETE_FILES_PERIOD_MICRO_SECONDS_DEFAULT =
```

File: hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/DatanodeConfiguration.java [hunk 3]
```diff
@@ -72,6 +73,16 @@ public class DatanodeConfiguration {
...
+      6 * 60 * 60 * 1000 * 1000;
+  public static final String ROCKSDB_LOG_MAX_FILE_SIZE_KEY =
```
Comment (kerneltime) [inline hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/DatanodeConfiguration.java:80]:
```suggestion
  public static final String ROCKSDB_LOG_MAX_FILE_BYTES_SIZE_KEY =
```

**Review by kerneltime — APPROVED** [review]
Minor nits.

File: hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/DatanodeConfiguration.java (hunk 2 above)
Comment (ChenSammi) [reply hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/DatanodeConfiguration.java:78]: Yes, it's micro seconds.

**Pull request merged**